server/meshmodel/components/core/v1.0.0/Application.json

```diff
@@ -0,0 +1,24 @@
+{
+	"kind": "Application",
```

Comment:
Whoa... how'd this sneak in here? 😉

Reply:
What happened to the other PR to unit this?

Reply:
The component generator may have deleted that, I am checking logic right now to make sure that existing models are not deleted.